datafusion/physical-expr/src/planner.rs

```diff
@@ -114,22 +115,11 @@ pub fn create_physical_expr(
     match e {
         Expr::Alias(Alias { expr, metadata, .. }) => {
             if let Expr::Literal(v, prior_metadata) = expr.as_ref() {
-                let mut new_metadata = prior_metadata
-                    .as_ref()
-                    .map(|m| {
-                        m.iter()
-                            .map(|(k, v)| (k.clone(), v.clone()))
-                            .collect::<HashMap<String, String>>()
-                    })
-                    .unwrap_or_default();
-                if let Some(metadata) = metadata {
-                    new_metadata.extend(metadata.clone());
-                }
-                let new_metadata = match new_metadata.is_empty() {
-                    true => None,
-                    false => Some(new_metadata),
-                };
-
+                let metadata = metadata.as_ref().map(|m| FieldMetadata::from(m.clone()));
```

Comment:
I remove this clone in a follow on PR:
- https://github.com/apache/datafusion/pull/16320